minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/RunMiNiFi.java

```diff
@@ -1197,7 +1197,14 @@ public void start() throws IOException, InterruptedException {
         this.changeListener = new MiNiFiConfigurationChangeListener(this, defaultLogger);
         this.periodicStatusReporters = initializePeriodicNotifiers();
         startPeriodicNotifiers();
-        this.changeCoordinator = initializeNotifier(this.changeListener);
+        try {
+            this.changeCoordinator = initializeNotifier(this.changeListener);
+        } catch (Exception e) {
```

Comment:
As this is effectively our main, I am okay with this as this is our last stop before the stacktrace reaches user land.  In our current state this throws a RuntimeException which causes the JVM to exit as it bubbles up.  From my perspective, we cannot enumerate all possibilities at this point nor is there significant gain to do so and would potentially be bringing dependencies to this class/module.  We could optionally wrap this with another Exception class, but that doesn't seem to buy us much either.  Let me know your thoughts.